### PR TITLE
Port to starlark

### PR DESCRIPTION
**Pull request by dschmidt**

### CODE REVIEW

**Comment by dschmidt** [comment]
This also removes 2.4 client and adds 2.6.

**Comment by dschmidt** [comment]
May not all of the pipelines need to build the client themselves, that would save us quite a bit of execution time as well. But that's out of scope for this PR.

File: .drone.star [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                    # "sha256": server_version["sha256"],
+                    "destination": "owncloud.tar.bz2",
+                },
+            },
+            {
+                "name": "extract",
+                "image": "owncloud/ubuntu:latest",
+                "pull": "always",
+                "commands": [
+                    "tar -xjf owncloud.tar.bz2 -C /var/www",
```
Comment (tboerger) [inline .drone.star]: This will be lost for the next step.

Comment (dschmidt) [reply .drone.star]: Huh?! Why would it? `/var/www` is the workspace. And as you can see in drone, this builds just fine.

Comment (tboerger) [reply .drone.star]: Oh... you are setting a workspace...

Comment (dschmidt) [reply .drone.star]: Yeah, I basically ported everything as close to the original as possible while porting from 0.8 to 1.0. Please don't make refactor anything here that wasn't better before ;-)

**Pull request merged**